server/utils/resident/index.ts

```diff
@@ -43,3 +45,51 @@ export const residentTabItems = (placement: Cas1SpaceBooking, activeTab: Residen
     href: getSelfLink(key as ResidentProfileTab),
   }))
 }
+
+export function getResidentHeader(placement: Cas1SpaceBooking) {
+  const person = placement.person as FullPerson
+
+  return {
+    name: person.name,
+    photoUrl: '/assets/images/resident-placeholder.png',
+    badges: [
+      getBadge('Unknown RoSH', 'red'),
+      getBadge('Unknown MAPPA', 'purple'),
+      getBadge('Unknown ', 'black'),
+      getBadge('Unknown 2', 'black'),
+      getBadge('Unknown 3', 'black'),
+      '<span><a href="#">+3 risk flags</a></span>',
+    ],
+    attributes: [
+      [
+        { title: 'CRN', description: person.crn },
+        { title: 'Approved Premises', description: placement.premises.name },
+        { title: 'Key worker', description: placement.keyWorkerAllocation?.name ?? 'Not assigned' },
+      ],
+      [
+        {
+          title: 'Arrival',
+          description: DateFormats.isoDateToUIDate(placement.expectedArrivalDate, { format: 'short' }),
```

Comment:
This should use the canonical date fn.  I left this in the controller, but would be better here:
`const { arrivalDate, departureDate } = canonicalDates(placement)`

Likewise for departure date below.